disnake/client.py

```diff
@@ -768,6 +785,11 @@ async def connect(self, *, reconnect: bool = True) -> None:
         _, gateway, session_start_limit = await self.http.get_bot_gateway()
         self.session_start_limit = SessionStartLimit(session_start_limit)
 
+        if not ignore_session_start_limit and self.session_start_limit.remaining == 0:
+            raise SessionStartLimitReached(
+                f"Daily session start limit has been reached, resets at {self.session_start_limit.reset_time}"
```

Comment:
message should be moved into the exception's `__init__`

Reply:
https://github.com/DisnakeDev/disnake/pull/537/commits/126f9b89c9c0d8443100b673215c472da34c8e03